### DISCUSSION TITLE
How can I make the dark theme a bit better?

### DISCUSSION THREAD

**Discussion by gerroon** [ggozad/oterm#129] [Q&A] [answered]
Hi

The dark theme is  nice but it has these light elements that is making it hard to look at. Is there a way to make those gray boxes go away?

![image](https://github.com/user-attachments/assets/4ff4e391-9512-45d5-8285-8d7c34e9dfe2)

**Comment by ggozad** [comment]
Hey @gerroon!
Sorry for the very late reply. I think your terminal does not seem to support color schemes. What terminal are you using? See the screenshots for how it's "supposed" to look!

**Reply by gerroon** [reply]
I am using Ubuntu under WSL, I am not sure how that dynamic works since it runs under Cmd.exe

**Comment by ggozad** [comment]
I am not really that familiar with windows terminals, but I have tried it with the default ubuntu terminal in wsl and it worked fine. Iirc that is different from `cmd.exe`. Powershell might also work.

**Reply by gerroon** [reply]
Powershell has similar issues but oterm crashes with this message when I run it under Powershell which is weird "FileNotFoundError: [Errno 2] No such file or directory: 'clip.exe' " This happens if I click in any area.

Is there a way to run it with monocrome two colors only? 

What I have under Powershell
![image](https://github.com/user-attachments/assets/1ef03532-df99-40c5-8027-1a26633ddbd1)

**Comment by ggozad** [comment]
I would go with the Ubuntu terminal then... II am afraid only the default themes from `textual` are available.

**Comment by gerroon** [comment, marked as answer]
For those who have a similar problem, try running Ubuntu under Tabby. It is much better.
